lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -21,8 +23,8 @@ const publicReadBuckets = process.env.ALLOW_PUBLIC_READ_BUCKETS ?
 
 function checkBucketAcls(bucket, requestType, canonicalID, mainApiCall) {
     // Same logic applies on the Versioned APIs, so let's simplify it.
-    const requestTypeParsed = requestType.endsWith('Version') ?
-        requestType.slice(0, 'Version'.length * -1) : requestType;
+    const requestTypeParsed = requestType.endsWith('Version')
+        ? requestType.slice(0, -7) : requestType;
```

Comment:
Why are you changeing that ?

Reply:
Good catch , I'm going to use the version.length indeed

Reply:
done here : https://github.com/scality/cloudserver/pull/5432/commits/af1b7feaa4485ae0a978b06b1be60c1f31884367#diff-773344fcfc0f64145ff95eb6eccd556588c19449dc2c263a2532d4030bf00dffR27